language/google/cloud/language/entity.py

```diff
@@ -176,4 +184,7 @@ def from_api_repr(cls, payload):
         metadata = payload['metadata']
         salience = payload['salience']
         mentions = [Mention.from_api_repr(val) for val in payload['mentions']]
-        return cls(name, entity_type, metadata, salience, mentions)
+        sentiment = None
+        if payload.get('sentiment'):
```

Comment:
Why would there be an empty dictionary in there? The JSON payload dumps any empty fields (as does the protobuf payload)